lib/phoenix/router/scope.ex

```diff
@@ -113,7 +114,8 @@ defmodule Phoenix.Router.Scope do
       pipes: top.pipes,
       private: Map.merge(top.private, private),
       assigns: Map.merge(top.assigns, assigns),
-      log: Keyword.get(opts, :log, top.log)
+      log: Keyword.get(opts, :log, top.log),
+      trailing_slash?: Keyword.get(opts, :trailing_slash, top.trailing_slash?) == true
```

Comment:
I'm not sure if this is correct. Nested scopes don't appear to get the option from their parent scope, so maybe I'm not targeting something correctly for the default?

Reply:
FWIW it looks correct to me. Are you seeing something that says differently? Do you have a failing test?

Reply:
Not yet, but I'll add one to be sure.